### ISSUE TITLE
[doc] 8.5  Locking Installed Program Temporary Fixes

### ISSUE THREAD

**Issue by asemen** [SUSE/doc-caasp#124]
8.5  Locking Installed Program Temporary Fixes

http://docserv.suse.de/documents/CaaS_Platform_3/caasp-admin/single-html/#sec.admin.troubleshooting.velum_ptf

There is a problem that the comand to install PTF do not work:
On run of:
 1. Install the PTF manually.
root: transactional-update reboot PTF_FILE.rpm install

I do get 
 transactional-update reboot *.rpm install
Usage: transactional-update --help|--version
       transactional-update [--no-selfupdate] [cleanup][up|dup|patch|initrd][kdump][reboot]
       transactional-update [--no-selfupdate] [cleanup] [reboot] pkg install|remove|update PKG1..PKGN
       transactional-update [--no-selfupdate] migration
       transactional-update [--no-selfupdate] register -p <product> [-r <registration code>]
       transactional-update rollback [number]

But I can run:
transactional-update pkg install *TEST*rpm

So this have to be changed in my opinion.

transactional-update pkg install *PTF*rpm

**Comment by r0ckarong** [comment]
Yes the way this usage of transactional-update works has changed. I will correct the troubleshooting section.

**Comment by r0ckarong** [comment]
Added with https://github.com/SUSE/doc-caasp/commit/c9cd328f23953fef400aa447000e7421ffaee28e
